lib/model/query/datasets.js

```diff
@@ -121,9 +121,14 @@ const _getByIdSql = ((fields, datasetId) => sql`
 
 const _getByNameSql = ((fields, datasetName, projectId, includeForms) => sql`
     SELECT
-        ${fields}
+        ${fields}, stats."lastEntity" "datasets!lastEntity"
     FROM
         datasets
+    LEFT OUTER JOIN (
+      SELECT "datasetId", MAX(COALESCE("updatedAt", "createdAt")) "lastEntity" 
```

Comment:
Should have edited my comment rather than deleting. `MAX(COALESCE("updatedAt", "createdAt"))` wasn't looking right to me but I have figured it out.